inc/classes/class_display.php

```diff
@@ -136,142 +204,99 @@ public function AddHeaderMenu($names, $link, $active = null)
 
         $MainContent .= $items;
     }
-    
+
+    /**
+     * @param array     $names
+     * @param string    $link
+     * @param string    $active
+     * @return void
+     */
     public function AddHeaderMenu2($names, $link, $active = null)
     {
-        global $templ, $MainContent;
+        global $MainContent;
 
+        $items = '';
         foreach ($names as $key => $name) {
-            ($key == $active and $active != '')? $am = '' : $am = 'class="menu"';
+            if ($key == $active && $active != '') {
+                $am = '';
+            } else {
+                $am = 'class="menu"';
+            }
             $items .= '<a href="'. $link . $key .'"'. $am .'><b>'. $name .'</b></a> - ';
         }
         $items = substr($items, 0, -3);
-    
+
         $MainContent .=  $items;
     }
-  
-  // dynamic = 1 --> Jquery-Javascript-Tabs
-  // dynamic = 0 --> static tabs without Jquery & Javascript
-    public function AddJQueryTabsStart($id, $dynamic = null)
```

Comment:
I was surprised to see that this is considered dead code, as I saw tabs being used in multiple locations.
But it seems that this was now covered by `StartTabs()`and `StartTab()` and a bit of JS-Code in `scripts.js`